### PR TITLE
completed forms exercise - Nolan Reda

### PR DESCRIPTION
**Pull request by NolanReda**
form

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Nolan! Two notes:
1. You should not make use of `br` elements, they are outdated. Instead, make any element that needs to go to a new line the child of a `div` element.
2. You are missing `label` elements entirely, please revise this to use `label` elements for text that is labelling a specific form control element.

**Comment by NolanReda** [comment]
form

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+<html>
+  <head>
+    <title>Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <label for="email">Email</label>
+      <div>
+        <input type="text" name="email" size="25" required placeholder="Email">
```
Comment (codingcodymiller) [inline html-forms/index.html:11]: This `input` needs an `id` to match with the `for` attribute on its `label`

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      </div>
+      <label for="why-is-coding-cool">Why is coding cool?</label>
+      <div>
+        <select name="Select">
```
Comment (codingcodymiller) [inline html-forms/index.html:15]: This `select` needs an `id` to match with the `for` attribute on its `label`

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+          <option value="Interesting">Because it's interesting</option>
+          <option value="Fun">Because it's fun</option>
+          <option value="Powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <p>Favorite Languages(choose one)</p>
+      <div>
+        <input type="radio" name="language" value="HTML"> HTML
+        <input type="radio" name="language" value="CSS"> CSS
+        <input type="radio" name="language" value="JavaScript"> JavaScript
```
Comment (codingcodymiller) [inline html-forms/index.html:26]: The text next to each of these radio inputs should be in a `label`

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+      </div>
+      <p>Would you like to sign up for more information?</p>
+      <div>
+        <input type="checkbox" name="sign up" value="Sign me up!"> Sign me up!
```
Comment (codingcodymiller) [inline html-forms/index.html:30]: The text next to this checkbox should be in a `label`

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+      </div>
+      <div>
+        <label for="comments">Comments</label>
+      </div>
+      <div>
+        <textarea name="comments" cols="33" rows="10"></textarea>
```
Comment (codingcodymiller) [inline html-forms/index.html:36]: This `textarea` needs an `id` to match with the `for` attribute on its `label`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-forms/index.html (hunk 1 above)
Comment (NolanReda) [reply html-forms/index.html:11]: do you mean a separate `id` element?

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**